File: FrontendTemplateProject-main/src/app/models/Calendar-Tim/eventDTO.ts
import { Guid } from 'guid-typescript';

export class eventDTO {
    
    eventId!: Guid;
  eventName!: String;
  eventDate!: Date;
  eventTime!: Date;
  eventDescription!: String;
  eventTypeId!: Guid;
  calendarId!: Guid;
    
    constructor(
        eventId: Guid,
        eventName: String,
        eventDate: Date,
        eventTime: Date,
        eventDescription: String,
        eventTypeId: Guid,
        calendarId: Guid,
    ) {
        this.eventId = eventId;
        this.eventName = eventName;
        this.eventDate = eventDate;
        this.eventTime = eventTime;
        this.eventDescription = eventDescription;
        this.eventTypeId = eventTypeId;
        this.calendarId = calendarId;
    }
    
}

File: Grupa11-Calendar/Grupa11-Calendar/DTO/EventDTO.cs
namespace Grupa11_Calendar.DTO
{
    public class eventDTO
    {
        #region Event 
        public string EventName { get; set; }
        public DateTime EventDate { get; set; }
        public DateTime EventTime { get; set; }
        public string EventDescription { get; set; }
        public Guid EventTypeId { get; set; }
        public Guid CalendarId { get; set; }
        #endregion
    }
}
